src/mantarray_desktop_app/utils.py

```diff
@@ -406,16 +405,10 @@ def _create_start_recording_command(
     if isinstance(barcode, str):
         are_barcodes_matching = _check_scanned_barcode_vs_user_value(barcode, shared_values_dict)
 
-    if is_calibration_recording:
-        customer_account_id = shared_values_dict["config_settings"].get(
-            "customer_account_id", NOT_APPLICABLE_H5_METADATA
-        )
-        user_account_id = shared_values_dict["config_settings"].get(
-            "user_account_id", NOT_APPLICABLE_H5_METADATA
-        )
-    else:
-        customer_account_id = shared_values_dict["config_settings"]["customer_account_id"]
-        user_account_id = shared_values_dict["config_settings"]["user_account_id"]
+    customer_account_id = shared_values_dict["config_settings"].get(
+        "customer_account_id", NOT_APPLICABLE_H5_METADATA
+    )
+    user_account_id = shared_values_dict["config_settings"].get("user_account_id", NOT_APPLICABLE_H5_METADATA)
```

Comment:
Removed because customer/user accounts are no longer required in order to make a recording